src/Tasks/Common/Resources/Strings.resx

```diff
@@ -758,4 +758,20 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1163: Cannot emit symbols when publishing for .NET 5 with Crossgen2 using composite mode.</value>
     <comment>{StrBegin="NETSDK1163: "}</comment>
   </data>
-</root>
+  <data name="DuplicateTypeLibraryIds" xml:space="preserve">
+    <value>NETSDK1164: The same resource id {0} was specified for two type libraries '{1}' and '{2}'. Duplicate type library ids are not allowed.</value>
```

Comment:
~~id~~ -> ID

I lost this battle years ago - ID is the correct casing for ID and id is the buddy of the ego. I only accept this argument in comments and public documentation. Everywhere else it is up to you :-)

Reply:
The next two messages should also be updated.